docs/provider/conjur.md

```diff
@@ -93,20 +117,34 @@ You can use an external JWT issuer or the Kubernetes API server to create the to
 kubectl create token my-service-account --audience='https://conjur.company.com' --duration=3600s
 ```
 
-Save the secret store file as `conjur-secret-store.yaml` (the filename used in subsequent steps).
+Save the secret store file as `conjur-secret-store.yaml`.
+
+##### Step 2: Create the external secrets store
+
+```shell
+# WARNING: creates the store in the "external-secrets" namespace, update the value as needed
+#
+kubectl apply -n external-secrets -f conjur-secret-store.yaml
+
+# WARNING: running the delete command will delete the secret store configuration
+#
+# If there is a need to delete the external secretstore
+# kubectl delete secretstore -n external-secrets conjur
+```
+
+### Define an external secret
 
-#### Step 2: Define an external secret
+Now that you have set up the Conjur provider secret store, you can define an external secret that will fetch one or more secrets from Conjur.
```

Comment:
`After you have configured the Conjur provider secret store, you can fetch secrets from Conjur.`